xml/System.Text.Json/JsonElement.xml

```diff
@@ -163,7 +163,9 @@
       <Docs>
         <summary>Gets the current JSON number as a <see cref="T:System.Double" />.</summary>
         <returns>The current JSON number as a <see cref="T:System.Double" />.</returns>
-        <remarks>To be added.</remarks>
+        <remarks>This method does not parse the contents of a JSON string value.On .NET Core this method returns <see cref="F:System.Double.PositiveInfinity" /> (or
```

Comment:
Several of the paragraphs seem to be glued together (e.g. "value.On" and later in the diff "comparison.If")

Reply:
There were some unexpected endlines that I had to alter manually before commiting. It seems I missed a few. Thanks for finding them, I'll fix them.

Reply:
Fixed!